CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ Main (unreleased)
     (@marctc).
   - `prometheus.exporter.mysql` collects metrics from a MySQL database.
     (@spartan0x117)
+  - `otelcol.auth.sigv4` performs `sigv4` authentication for making requests to AWS services 
```

Comment:
Can we have this align with the documentation reference and initially spell this out as "AWS Signature Version 4 (SigV4)" for the first time it's referenced, and then use "SigV4" for other references?